electron-builder.json

```diff
@@ -8,6 +8,13 @@
   "afterSign": "scripts/after-sign.js",
   "asarUnpack": ["**/*.node"],
   "files": ["**/*", "!**/node_modules/@bitwarden/desktop-native/**/*"],
+  "publish": {
```

Comment:
We need to define `endpoint` here https://www.electron.build/configuration/publish, otherwise it will attempt to fetch it directly from S3 which we don't want.

Reply:
Hey thanks for the reminder. I just missed that one 👍 .